2020-Contributor-Survey.md

```diff
@@ -0,0 +1,124 @@
+# DRAFT -- 2020 Node.js Contributor Survey
+
+* How long have you been contributing to Node.js?
+
+  * Less than one year
+  * One to two years
+  * Two to three years
+  * Three to five years
+  * Five+ years
+
+* What kind of contributor do you consider yourself to be?
+
+  * I'm not yet a contributor
+  * I've contributed once or twice
+  * I occasionally contribute
+  * I regularly contribute
+  * I've contributed substantially
+  * I used to contribute but no longer do
+
+* Over the past year, do you consider your contributions to have:
+
+  * Increased
+  * Decreased
+  * Remained about the same
+
+* In the coming year, do you expect your contributions to:
+
+  * Increase
+  * Decrease
+  * Remain the same
+  * I have no idea
+
+* What region of the world are you in?
+
+* How many other open source projects not under the Node.js project do you contribute to?
+
+  * This is my first open source project!
+  * 1 other
+  * 2 or more
```

Comment:
this is a broad category :-) maybe add one more tranche?

Reply:
Heh... I was thinking about dropping this question. Not sure it adds much value. I think a more compelling question would be whether the individual contributes to other projects in the ecosystem.